### PR TITLE
[Appsource][Content-update][1570501]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Updated content. Please review,

### CODE REVIEW

File: msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md [hunk 1]
```diff
@@ -32,16 +32,15 @@ When a user installs your application one of the first things they do is consent
...
->
+> * A bot is added or removed.
+> * An existing bot's unique `botId` value is changed.
+> * An existing bot's `isNotificationOnly` property is changed.
+> * An existing bot's `supportsFiles` `supportsCalling` properties are changed.
+> * A messaging extension `composeExtensions` is added or removed.
+> * A new connector is added.
+> * A new static or personal tab is added.
+> * A new configurable group or channel tab is added.
+> * The properties inside `webApplicationInfo` are changed. For changes to `webApplicationInfo`, consent is only required in Team scope.
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md:43]:
```suggestion
> * A bot is added or removed.
> * An existing bot's unique `botId` value is changed.
> * An existing bot's `isNotificationOnly` boolean value is changed.
> * An existing bot's `supportsFiles` and `supportsCalling` boolean values are changed.
> * A messaging extension `composeExtensions` is added or removed.
> * A new connector is added.
> * A new static or personal tab is added.
> * A new configurable group or channel tab is added.
> * The properties inside `webApplicationInfo` are changed. For changes to `webApplicationInfo`, consent is only required in the Teams scope.
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Rajeshwari-v - please validate and commit the changes.

File: msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md [hunk 2]
```diff
@@ -32,16 +32,15 @@ When a user installs your application one of the first things they do is consent
...
-> * A messaging extension (`composeExtensions`) was added or removed.
-> * A new connector was added.
-> * A new static/personal tab was added.
-> * A new configurable group/channel tab was added.
-> * The `webApplicationInfo` values changed.
->
+> * A bot is added or removed.
+> * An existing bot's unique `botId` value is changed.
+> * An existing bot's `isNotificationOnly` boolean value is changed.
+> * An existing bot's `supportsFiles` and `supportsCalling` boolean values are changed.
```
Comment (Wajeed-msft) [inline msteams-platform/concepts/deploy-and-publish/appsource/post-publish/overview.md:38]: Could you please change it to or? 

`supportsFiles` **or** `supportsCalling`

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approved the PR

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**